Makefile.pre.in

```diff
@@ -1037,6 +1039,9 @@ buildbottest:	build_all platform
 		fi
 		$(TESTRUNNER) -j 1 -u all -W --slowest --fail-env-changed --timeout=$(TESTTIMEOUT) $(TESTOPTS)
 
+pythoninfo: build_all
```

Comment:
I hesitate between "build_all" and "$(BUILDPYTHON)".

Using build_all, "make pythoninfo" runs again setup.py to try to build extensions, as done by "make buildbottest". I expect that running setup.py multiple times has no side effect.

Using $(BUILDPYTHON), "make pythoninfo" fails from a fresh build because basic modules like _posixsubprocess are not built yet.